frappe/patches/v14_0/update_attachment_comment.py

```diff
@@ -0,0 +1,31 @@
+import frappe
+
+
+def execute():
+	# Strip everything except link to attachment and icon from comments of type "Attached"
+	for name, content in frappe.get_all(
```

Comment:
On old sites, this can fail if it goes beyond 200K queries. 

Enable `frappe.db.auto_commit_on_many_writes`